sdk/spring/azure-spring-boot-starter-metrics/README.md

```diff
@@ -11,15 +13,13 @@ With this starter you could easily use Azure Metrics with Spring Boot.
 - [Maven](http://maven.apache.org/) 3.0 and above
 
 ### Include the package
-[//]: # ({x-version-update-start;com.azure.spring:azure-spring-boot-starter-metrics;current})
```

Comment:
We should have a further discussion on this, should we still keep the including part? We didn't include this block in this [doc](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/spring/azure-spring-boot-starter-mediaservices).

Reply:
BTW, the doc I paste doesn't render correctly (the title), please change it in this PR.